### ISSUE TITLE
SNMP stats poller crashes when Realtime Replication is enabled to multiple clusters

### ISSUE THREAD

**Issue by nerophon** [basho/riak_ee-issues#23]
When Realtime Replication is enabled to multiple clusters and SNMP is being used, a crash occurs in the SNMP stats poller that looks like the following:

```
[error] <0.6738.0> poll_stats: error function_clause @ [
{riak_snmp_stat_poller,'-set_rows/4-fun-0-',["cluster1","cluster2"],
[{file,"src/riak_snmp_stat_poller.erl"},{line,533}]} ...
```

This is a public posting of an internal issue:
https://github.com/basho/riak_snmp/issues/27

Recommended workaround until fixed: use the `/riak-repl/stats` HTTP endpoint.

**Comment by nerophon** [comment]
This issue was fixed in version 2.1.2.

**Comment by nerophon** [comment]
Further workaround:

You can change the SNMP poll rate, and thereby the frequency of the log messages, by setting `snmp.refresh_frequency` in `riak.conf`. For example to set an interval of 1 day use:

```
snmp.refresh_frequency=1d
```

It can also be changed without restarting on a single node via `riak attach` with

```
riak_core_util:rpc_every_member_ann(application,set_env,[riak_snmp, polling_interval, 86400000], 5000).
```

You can replace `86400000` with the desired interval in milliseconds. As usual, exit from the Erlang shell with `Ctrl+G q Return`
